src/Widgets/calendar/Grid.vala

```diff
@@ -40,6 +41,7 @@ namespace DateTime.Widgets {
         private Gtk.Revealer[] week_labels;
 
         construct {
+            model = Widgets.CalendarModel.get_default ();
```

Comment:
If we want the model to be static, probably we should create it in a `static construct {}` block